### DISCUSSION TITLE
you are GALAXY BIG 🧠

### DISCUSSION THREAD

**Discussion by pranjal779** [Schweinepriester/github-profile-achievements#607] [Q&A] [answered]
galaxy-brain badge 💪 🧠

**Comment by Spectrum5** [comment, marked as answer]
thanks

**Reply by pranjal779** [reply]
tell me if you get the way to dank big brain achievement

**Reply by Spectrum5** [reply]
Yup I got it.. Thanks buddy =)

**Reply by pranjal779** [reply]
do you get it?

**Comment by dee7ine** [comment]
Is it the place to become Galaxy Brain?

**Reply by pranjal779** [reply]
yes, but @Spectrum5 already got answered.
do you need it?

**Reply by dee7ine** [reply]
Yes, my bad, I'm new to github discussions :P
